cmd/prstatus/prstatus.go

```diff
@@ -88,7 +88,7 @@ func run(c *cobra.Command, _ []string) {
 	statuses := make(map[string]int)
 	reactions := make(map[string]int)
 
-	detailsTable := table.New("Repository", "State", "Reviews", "URL")
+	detailsTable := table.New("Repository", "State", "Reviews", "Build status", "URL")
```

Comment:
Should we be calling it `Mergeable` so that we're consistent with GitHub's terminology?

Reply:
Hey Richard, I changed the approach because I thought `Mergeable` was an aggregation of the build steps but turns out it isn't and it just checks if there are conflicts or not.

I pushed some changes and I'm using now a field called `statusCheckRollup` which is an array containing all the checks for the PR and their individual state

Reply:
Changed it to `Checks status`